fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +220,36 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+Mask=`opts_GetOpt1 "--usemask" $@`                                   # what mask to use for the final bold (T1: the default, BOLD: one made on the reference bold image, DILATED: dilated MNI mask, NONE:none)
+
+# Defaults
+Mask=${Mask:-T1}
+
+# ------------------------------------------------------------------------------
+#  Compliance check
+# ------------------------------------------------------------------------------
+
+MPPMode=`opts_GetOpt1 "--mpp-mode" $@`
+MPPMode=`opts_DefaultOpt $MPPMode "HCPStyleData"`
+Compliance="HCPStyleData"
+ComplianceMsg=""
+
+if [ ! "${Mask}" = 'T1' ]; then
```

Comment:
I think the purpose of the mask is more to reduce file sizes by setting non-brain to exact 0s, so I'm not sure this needs to be restricted to legacy data mode.

Reply:
Oh right, for intensity normalization...depending on how we do that, it may be important to use a tight mask.

Reply:
I'm not sure if there is a way to compute the intensity normalization using a tighter mask and then apply it on a wider one.